spec/views/users_show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
```

Comment:
- Hi, I would advise you to check the test you wrote on your end before submitting it. All the tests are falling.
![Capture](https://user-images.githubusercontent.com/63972670/217497967-e4ffbe19-0017-4268-a882-2055874f2ad8.JPG)
**NB: All files in spec views folder did not pass.**